joy2twist/src/joy2twist_node.cpp

```diff
@@ -0,0 +1,91 @@
+#include "joy2twist/joy2twist_node.hpp"
+
+namespace joy2twist
+{
+Joy2TwistNode::Joy2TwistNode() : Node("joy2twist_node")
+{
+  using namespace std::placeholders;
+
+  declare_parameters();
+  load_parameters();
+
+  auto sensor_qos = rclcpp::QoS(
+    rclcpp::QoSInitialization(
+      rmw_qos_profile_sensor_data.history, rmw_qos_profile_sensor_data.depth),
+    rmw_qos_profile_sensor_data);
+
+  joy_sub_ =
+    create_subscription<MsgJoy>("joy", sensor_qos, std::bind(&Joy2TwistNode::joy_cb, this, _1));
+  twist_pub_ = create_publisher<MsgTwist>("cmd_vel", sensor_qos);
+
+  RCLCPP_INFO(get_logger(), "Joy2Twist node is initialized");
```

Comment:
Is it necessary to point it is Joy2Twist node if it will be written there anyway in a INFO message?

Reply:
Done ✅